examples/apache-basic-playbook/templates/httpd.conf.j2

```diff
@@ -165,8 +165,8 @@ DocumentRoot "/var/www/html"
 </IfModule>
 
 #
-# The following lines prevent .htaccess and .htpasswd files from being 
-# viewed by Web clients. 
+# The following lines prevent .htaccess and .htpasswd files from being
+# viewed by Web clients.
```

Comment:
Below this around line 183 (Github won't let me leave a comment on a line you didn't change) there is a `MaxKeepAliveRequests {{ apache_max_keep_alive_requests }}` in this template that wasn't removed and is giving me an error in my tests.